### ISSUE TITLE
Redis doc contains error discription

### ISSUE THREAD

**Issue by zonghaishang** [apache/incubator-dubbo-website#158]
```
2. 服务消费方启动时，从 Channel:/dubbo/com.foo.BarService/providers 订阅 register 和 unregister 事件
3.并向 Key:/dubbo/com.foo.BarService/providers 下，添加当前消费者的地址
```

should be change to :

```
2. 服务消费方启动时，从 Channel:/dubbo/com.foo.BarService/providers 订阅 register 和 unregister 事件
3.并向 Key:/dubbo/com.foo.BarService/consumers 下，添加当前消费者的地址
```

**Comment by ralf0131** [comment]
Hi, would you please send a pull request?

**Comment by kexianjun** [comment]
fixed in #270
